galaxy_ng/app/api/v3/serializers/task.py

```diff
@@ -1,15 +1,55 @@
 import logging
-
+from typing import Any, Dict, Optional
 from rest_framework import serializers
+from drf_spectacular.utils import extend_schema_field
+from pulpcore.app.serializers import ProgressReportSerializer
```

Comment:
I am getting this error

```
ERROR: Detected bad imports from pulpcore:
./galaxy_ng/app/api/v3/serializers/task.py:5:from pulpcore.app.serializers import ProgressReportSerializer
```

So probably we need to have `ProgressReportSerializer` exposed via `pulpcore.plugin` what do you think @bmbouter  @daviddavis @newswangerd ?

Reply:
+1 from me, @rochacbruno could you open a PR against pulpcore for exposing this?

Reply:
https://github.com/pulp/pulpcore/pull/982

Reply:
Thanks I commented w/ a small request. Note this won't be able to be released until 3.9.0 which we don't have a scheduled date yet but it's usually 4ish weeks from the last one which was Oct 20.